### PR TITLE
changed data-offset to 1 as it was off by 1 pixel

### PR DESCRIPTION
**Pull request by jusung1459**

### CODE REVIEW

File: mainsite/assets/js/jquery-func.js [hunk 1]
```diff
@@ -7,12 +7,12 @@
     })
 
     $window.on('load', function () {
-        $body.scrollspy('refresh')
+        $.ready.then(function(){
```
Comment (modernNeo) [inline mainsite/assets/js/jquery-func.js]: why this change?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: mainsite/assets/js/jquery-func.js (hunk 1 above)
Comment (jusung1459) [reply mainsite/assets/js/jquery-func.js]: i forgot to take that out... Make a new pr?

Comment (modernNeo) [reply mainsite/assets/js/jquery-func.js]: just undo the change on your local copy of your branch and then do a push. the PR will automatically update when you do that,

**Review by modernNeo — APPROVED** [review]

**Comment by modernNeo** [comment]
looks good to me
